src/main/webapp/event-info-page/eventInfo.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB0s69v5OFVxuH0cN5IS96B3jwhnjB7tTo"></script>
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h2>Event Information</h2>
+    <span>Name:  </span> <span id="name">Cycle bun</span></br>
+    </br>
+    <span>Description:  </span><span id="description">Experience the burn of cycling drills! Take on cardiovascular conditioning with an interval-driven workout.</span></br>
+    </br>
+    <span>Name of Location:  </span><span id="location">Students Recreation Center</span></br>
```

Comment:
In the 1-to-1 weekly meeting yesterday, we thought that "Building Name:" might be a better choice to avoid confusion.